client/my-sites/checkout/src/components/cost-overrides-list.tsx

```diff
@@ -323,17 +324,20 @@ export function CouponCostOverride( {
 	const { formStatus } = useFormStatus();
 	const isDisabled = formStatus !== FormStatus.READY;
 	const isOnboardingAffiliateFlow = useSelector( getIsOnboardingAffiliateFlow );
+	const [ , experimentAssignment ] = useExperiment( 'calypso_checkout_hide_coupon_box' );
 
 	if ( ! responseCart.coupon || ! responseCart.coupon_savings_total_integer ) {
 		return null;
 	}
 
-	// translators: The label of the coupon line item in checkout, including the coupon code
+	// translators: The label of the coupon line item in checkout, including the coupon coden
+	const couponLabel = translate( 'Coupon: %(couponCode)s', {
+		args: { couponCode: responseCart.coupon },
+	} );
+
```

Comment:
The comment is part of the options object passed to the `translate` function and is not picked up from the code level `translators:` comment.
Ref: https://github.com/Automattic/wp-calypso/blob/trunk/packages/i18n-calypso/README.md#L32

This string has already been translated, so this is just a note for future reference. :)

Reply:
The translators comment was already there before my changes, It is highlighted because I move it 😃